sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -132,4 +132,8 @@ export enum NotificationMessageType {
    * ECE response file processing details.
    */
   ECEResponseFileProcessing = 11,
+  /**
+   * MSFAA record gets cancelled.
+   */
+  MSFAACancellationComplete = 12,
```

Comment:
What about `MSFAACancellation` for enum?

Reply:
Changed it. Thanks 👍